### DISCUSSION TITLE
Feedback: 0.19.10

### DISCUSSION THREAD

**Discussion by Arch-Shaman** [Arch-Shaman/ZK-Futurewars-Mod#223] [General]
trisolaran:
* Claims that RTG icon looks similar to pylon icon at a quick glance.
* Area Decloaker icon could be made a bit taller?

**Comment by Sharkgamedev** [comment]
> trisolaran:
> 
> * Claims that RTG icon looks similar to pylon icon at a quick glance.
> * Area Decloaker icon could be made a bit taller?

I see what he means about rtg icon. Any ideas on how to improve this?
